scripts/update_workflow.sh

```diff
@@ -0,0 +1,56 @@
+#  <workflow_name>
+
+# Step 1a - Export workflow
+java -jar config-slicer.jar \
+    -o custom \
+    -s <dev_server> \
+    -u <username> \
+    -p <password> \
+    -w '<workflow_name>' \
+    -m <workflow_name_without_space>_workflowManifest.txt
+
+
+# Step 1b - Export workflow
+java -jar config-slicer.jar \
+    -o export \
```

Comment:
potentially dangerous to use "export" in a bash script? (my knowledge is not great in bash)

Reply:
-o is an parameter which you have to set with one of the values custom / export / validate / import. So can not change that (is a tool from Illumina).